Packs/Gmail/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Gmail",
     "description": "Gmail API and user management (This integration replaces the Gmail functionality in the GoogleApps API and G Suite integration).",
     "support": "xsoar",
-    "currentVersion": "1.2.6",
+    "currentVersion": "1.2.7",
```

```suggestion
    "currentVersion": "1.3.0",
```

Please change the RN filename as well